### ISSUE TITLE
Suggestion for optional extensions (cd etc)

### ISSUE THREAD

**Issue by Busaruba2011** [Delta2Force/MCVmComputers#124]
If you wanted to, you could put a CD-ROM drive in your pc or something like a USB extension card.
With the CD-ROM, you could buy discs for iso's you have in the PC Shop.

I can make a CD-ROM drive design for you if you want.

**Comment by Delta2Force** [comment]
That’s true, you could maybe render the isos as burned cds with labels on them.

**Comment by Busaruba2011** [comment]
> That’s true, you could maybe render the isos as burned cds with labels on them.

👍👍👍
